test/tests.d/multilog_workflow.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+. vars
+
+#############                                                      ##############
+#       "multilog workflow"                                                     #
+#       test scenarios: behaviour of agent with the --multilog parameter        #
+#                                                                               #
+#############                                                      ##############
+
+
+HOST_MUTLIPLE_KEY=9df0ea6f-36fa-820f-a6bc-c97da8939a06
+
+# Reference: LOG-7549
+Scenario 'Agent follows files using --multilog parameter'
+
+Testcase 'Init'
+
+#$LE init --account-key=$ACCOUNT_KEY --host-key=$HOST_KEY --hostname myhost
+##e Initialized
+
+$LE init --account-key=$ACCOUNT_KEY --host-key=$HOST_MUTLIPLE_KEY
+#e Initialized
+
+Testcase 'Use --multilog parameter with pathname with no wildcard'
+
+mkdir apache-01
+touch apache-01/current
+
+$LE follow "$TMP/apache-01/current" --multilog
+#e Configuration files loaded: sandbox_config
```

Comment:
yes - there is a README in the /test folder - although mostly its about reading the existing test cases and the shell scripts that make it all work - the basic is the stderr or stdout is captured and compared using a diff against what is expected - the #e and #o are indicators of what to expect in each case